biz.aQute.bnd.exporters/src/aQute/bnd/exporter/subsystem/EasArchivType.java

```diff
@@ -0,0 +1,15 @@
+package aQute.bnd.exporter.subsystem;
+
+public enum EasArchivType {
+	NONE,
+	CONTENT,
+	ALL;
+
+	public static EasArchivType byParameter(String archiveContent) {
+
+		if (archiveContent == null) {
+			return CONTENT;/// default
+		}
+		return valueOf(archiveContent.toUpperCase());
```

Comment:
This will throw an exception if the input is not one of the enum values Perhaps returning null is better.